compiler/next/include/chpl/resolution/resolution-types.h

```diff
@@ -944,101 +977,28 @@ class FormalActualMap {
-  }
-};
-
-template<> struct stringify<chpl::resolution::ResolutionResultByPostorderID> {
-  void operator()(std::ostream &stringOut,
-                  StringifyKind stringKind,
-                  const chpl::resolution::ResolutionResultByPostorderID& stringMe) const {
-    stringOut << "resolution::ResolutionResultByPostorderID not stringified";
-  }
-};
-
-template<> struct stringify<chpl::resolution::ResolvedFunction> {
-  void operator()(std::ostream &stringOut,
-                  StringifyKind stringKind,
-                  const chpl::resolution::ResolvedFunction& stringMe) const {
-    stringOut << "resolution::ResolvedFunction not stringified";
-  }
-};
-
-
-template<> struct stringify<chpl::resolution::UntypedFnSignature>
-{
-  void operator()(std::ostream &stringOut,
-                  StringifyKind stringKind,
-                  const chpl::resolution::UntypedFnSignature& stringMe) const {
-    stringOut << "resolution::UntypedFnSignature not stringified";
-  }
-};
-
-
-template<> struct stringify<chpl::resolution::CallInfoActual>
-{
-  void operator()(std::ostream &stringOut,
-                  StringifyKind stringKind,
-                  const chpl::resolution::CallInfoActual& stringMe) const {
-    stringOut << "resolution::CallInfoActual not stringified";
-  }
-};
-
-template<> struct stringify<chpl::resolution::CallInfo>
-{
-  void operator()(std::ostream &stringOut,
-                  StringifyKind stringKind,
-                  const chpl::resolution::CallInfo& stringMe) const {
-    stringOut << "resolution::CallInfo not stringified";
-  }
-};
-
-template<> struct stringify<chpl::resolution::PoiInfo>
-{
-  void operator()(std::ostream &stringOut,
-                  StringifyKind stringKind,
-                  const chpl::resolution::PoiInfo& stringMe) const {
-    stringOut << "resolution::PoiInfo not stringified";
-  }
-};
 
 template<> struct stringify<chpl::resolution::TypedFnSignature::WhereClauseResult>
 {
   void operator()(std::ostream &stringOut,
                   StringifyKind stringKind,
                   const chpl::resolution::TypedFnSignature::WhereClauseResult& stringMe) const {
-    stringOut << "resolution::TypedFnSignature::WhereClauseResult not stringified";
-  }
-};
-
-template<> struct stringify<chpl::resolution::UntypedFnSignature::FormalDetail>
-{
- void operator()(std::ostream &stringOut,
-                 StringifyKind stringKind,
-                 const chpl::resolution::UntypedFnSignature::FormalDetail& stringMe) const {
-   stringOut << "resolution::UntypedFnSignature::FormalDetail not stringified";
- }
-};
-
-template<> struct stringify<chpl::resolution::TypedFnSignature>
-{
-  void operator()(std::ostream &stringOut,
-                  StringifyKind stringKind,
-                  const chpl::resolution::TypedFnSignature& stringMe) const {
-    stringOut << "resolution::TypedFnSignature not stringified";
-  }
+    std::string ret;
```

Comment:
let's make ret a `const char*` here to save an allocation. Also, it appears not to be used (presumably you need `stringOut << ret` at the bottom).